x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/bulk_actions/utils.ts

```diff
@@ -18,4 +18,5 @@ export const isIndexPatternsBulkEditAction = (editAction: BulkActionEditType) =>
     BulkActionEditTypeEnum.add_index_patterns,
     BulkActionEditTypeEnum.delete_index_patterns,
     BulkActionEditTypeEnum.set_index_patterns,
+    // @ts-expect-error upgrade typescript v4.9.5
   ].includes(editAction);
```

Comment:
It would be great to fix this error:

```ts
export const isIndexPatternsBulkEditAction = (editAction: BulkActionEditType) => {
  const indexPatternsActions: BulkActionEditType[] = [
    BulkActionEditTypeEnum.add_index_patterns,
    BulkActionEditTypeEnum.delete_index_patterns,
    BulkActionEditTypeEnum.set_index_patterns,
  ];
  return indexPatternsActions.includes(editAction);
};
```